src/object_properties.cpp

```diff
@@ -98,6 +100,9 @@ void ObjectProperties::serialize(std::ostream &os) const
 	writeF1000(os, automatic_face_movement_max_rotation_per_sec);
 	os << serializeString(infotext);
 	os << serializeString(wield_item);
+	writeV3F1000(os, selectionbox.MinEdge);
+	writeV3F1000(os, selectionbox.MaxEdge);
+	writeU8(os, !pointable);
```

Comment:
This can be fixed later without breaking anything so is there any chance one of you could give a second approval on this? It's fairly trivial stuff but useful for modders.